map/user_mark_container.cpp

```diff
@@ -173,8 +155,13 @@ void UserMarkContainer::Clear(size_t skipCount/* = 0*/)
 
 namespace
 {
-  static unique_ptr<PoiMarkPoint> s_selectionUserMark;
-  static unique_ptr<MyPositionMarkPoint> s_myPosition;
+static unique_ptr<PoiMarkPoint> s_selectionUserMark;
```

Comment:
What is the point to use static keyword here, for global variables declared in anonymous namespace? Also, could you please rename them in to g_selectionUserMark and g_myPosition?